LICENSE

```diff
@@ -254,13 +263,19 @@ https://github.com/elastic/elasticsearch-cloud-aws
 developed by Elastic
 License: Apache License v2 (http://www.apache.org/licenses)
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
 
 ================================================================================
 
 Meyer CSS reset
 http://meyerweb.com/eric/tools/css/reset/
 License: none (public domain)
 
+FIXME: I cannot find this source code in our Git repository. am I correct in
+thinking this is no longer used or is a build time inclusion?
+
```

Comment:
@mxm

Reply:
Not sure, maybe @m-kharbat knows about this one?

Reply:
oops sorry I meant to tag @m-kharbat!